addons/medical_gui/stringtable.xml

```diff
@@ -1179,5 +1179,13 @@
             <German>Schadensfarbe %1</German>
             <Polish>Obrażenia - kolor %1</Polish>
         </Key>
+        <Key ID="STR_ACE_Medical_GUI_ShowBloodEntry_DisplayName">
+            <English>Show Blood Loss</English>
+            <German>Blutverlust anzeigen</German>
+        </Key>
+        <Key ID="STR_ACE_Medical_GUI_ShowBloodlossEntry_Description">
+            <English>Show blood loss in the injury list.</English>
```

Comment:
I think including the "qualitative description" part from the comment in code may be an minor improvement.